adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala

```diff
@@ -308,7 +338,7 @@ class NarrowPeakParser extends FeatureParser {
 
     // skip empty or invalid lines
     if (fields.length < 3) {
-      if (log.isDebugEnabled) log.debug("Empty or invalid NarrowPeak line: {}", line)
+      log.warn("Empty or invalid NarrowPeak line: {}", line)
```

Comment:
Ah yeah, I wanted to make another pass back over this code and attach `ValidationStringency` flags and etc to this. How about I:

* Remove logging on blank/header lines
* Add a validation stringency flag:
  * Throw exception on malformed lines with strict stringency
  * log.warn on lenient
  * No logging on silent